src/main/java/zd/zero/waifu/motivator/plugin/WaifuMotivatorProject.java

```diff
@@ -56,6 +59,7 @@ public void projectOpened( @NotNull Project projectOpened ) {
             this.pluginState = WaifuMotivatorPluginState.getPluginState();
             this.unitTestListener = WaifuUnitTester.newInstance( projectOpened );
             this.idleEventListener = new IdleEventListener( projectOpened );
+            this.exitCodeListener = new ExitCodeListener( projectOpened );
```

Comment:
As we get more listeners to the project, we should start to think for a way to have a class to manage these listeners,  but for now, we still got a few listeners so we can still let this go